ui/eureka/FATEOpcodes.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This doesn't need to be kept in the ui/eureka/ folder, but I'm not sure on the best place. Thoughts welcome.

Reply:
I think `resources/fate_opcodes.json` might be a reasonable place, since we have a lot of other data there.